content/about/releases.en.md

```diff
@@ -11,4 +11,6 @@ After six months, odd-numbered releases (9, 11, etc.) become unsupported, and ev
 _LTS_ release status is "long-term support", which typically guarantees that critical bugs will be fixed for a total of 30 months.
 Production applications should only use _Active LTS_ or _Maintenance LTS_ releases.
 
+<Alert>The abbreviation LTS stands for `Long time support`.</Alert>
```

Comment:
It's actually long term support, not long time support.

Reply:
Oh ok